src/bundle.rs

```diff
@@ -57,6 +58,17 @@ pub unsafe trait Bundle: DynamicBundle {
         Self: Sized;
 }
 
+/// A dynamically typed collection of components that is also cloneable
+pub unsafe trait DynamicBundleClone: DynamicBundle + Clone {
```

Comment:
I think this is slightly more strict than necessary; we don't actually need to be able to clone the bundle itself, we just need to know how to clone its contents.
```suggestion
/// A dynamically typed collection of cloneable components
pub unsafe trait DynamicBundleClone: DynamicBundle {
```

Reply:
That is correct. Can't think of an implementation where those aren't transitive properties unless a custom implementation.